tests/integration/workflows/custom_make/test_custom_make.py

```diff
@@ -57,6 +57,48 @@ def test_must_build_python_project_through_makefile(self):
         output_files = set(os.listdir(self.artifacts_dir))
         self.assertEqual(expected_files, output_files)
 
+    def test_build_python_project_failed_through_makefile_without_working_directory(self):
+        source_code = os.path.join(os.path.dirname(__file__), "testdata", "makefile-in-different-working-directory")
+        manifest_path_valid = os.path.join(source_code, "Makefile")
+        with self.assertRaises(WorkflowFailedError):
```

Comment:
why does this test fail and raise a WorkflowFailedError? working_directory is an optional param right?

Reply:
+1

Reply:
ah I think it's because the requirements file is inside `source_code`, not in `makefile-in-different-working-directory`

Reply:
Yes, I had a call with Sriram to explain that but forgot to reply to this comment